### PR TITLE
Assign deliv emp to project feature

### PR DESCRIPTION
**Pull request by johnmcg23**
Fixed error to do with creating a delivery employee on previous commit.

### CODE REVIEW

**Review by UserConor — APPROVED** [review]
Class!

File: theLadsJava/db.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 user : JohnMc
-password:
-host:100.127.29.164
+password: John123
```
Comment (jkelso662) [inline theLadsJava/db.properties:2]: Low password strength 🙃

File: theLadsJava/src/main/java/org/kainos/ea/core/DelivEmployeeValidator.java [hunk 2]
```diff
@@ -14,12 +17,26 @@ public String isDelivEmployeeValid(DelivEmpRequest delivEmp){
...
         }
-        if (!delivEmp.getNiNum().matches("^[A-CEGHJ-PR-TW-Z][A-CEGHJ-NPR-TW-Z]\\d{6}[A-D]$")) {
-            return "National Insurance number is invalid.";
+        if(!isValidNINum(delivEmp.getNiNum())){
+            return "National Insurance number is not valid.";
         }
         return null;
     }
+
+    public boolean isValidNINum(String NINum){
```
Comment (jkelso662) [inline theLadsJava/src/main/java/org/kainos/ea/core/DelivEmployeeValidator.java:29]: Very nice refactoring! ❤️

File: theLadsJava/src/main/java/org/kainos/ea/resources/DelivEmpController.java [hunk 3]
```diff
@@ -99,5 +100,21 @@ public Response createDelivEmployee (@PathParam("id") int id, DelivEmpRequest de
         }
     }
 
+//    @POST
```
Comment (jkelso662) [inline theLadsJava/src/main/java/org/kainos/ea/resources/DelivEmpController.java]: Main should be kept clean. This code should live in another branch if it is WIP.

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Is this a bug fix PR? Not sure how all of this boilerplate code got into main. This is the first I'm seeing it! 🙃 Also your password cannot be merged into main!!!

**Pull request merged**